client/trino-client/src/main/java/io/trino/client/OkHttpUtil.java

```diff
@@ -328,4 +328,12 @@ public static void setupKerberos(
         clientBuilder.addInterceptor(handler);
```

Comment:
nit: improve commit message it wraps.

Reply:
Clipped message. let me know if you think it needs more information!

Reply:
It is way better. You can also copy the description from the pull request into it. That way it will be self explanatory.

Reply:
Missing description (from PR) in the commit message.

Reply:
Updated.